core/layouts/harris/harris.info

```diff
@@ -0,0 +1,19 @@
+name = Harris
+version = BACKDROP_VERSION
+backdrop = 1.x
+type = layout
+
+; Specify regions for this layout.
+regions[header]     = Header
+regions[top]        = Top
+regions[content]    = Content
+regions[sidebar]    = First Sidebar
+regions[sidebar2]   = Second Sidebar
```

Comment:
Also please note that `sidebar_first` and `sidebar_second` only appear in the `three_three_four_column` layout. All other layouts use only `sidebar` so the new proposed names are in fact more consistent with the old ones than `sidebar_first` and `sidebar_second`. :)